src/renderer.ts

```diff
@@ -596,7 +596,8 @@ const render = (format: Format, excludeStyles = false) =>
                 'data-duration': duration,
                 css: styles
             }
-            return styledH('div', attributes)
+            const figcaption = h(FigCaption, { format, caption: caption, credit: none });
+            return styledH('div', attributes, figcaption)
```

Comment:
I think `figcaption` should be a sibling rather than a child. Happy to go through this over chat.

Reply:
When the key/value is the same name in an object you can do 
```
{ format, caption, credit: none }
```

Reply:
> I think `figcaption` should be a sibling rather than a child so that it appears underneath the video

Yeah. In fact I think we should wrap both in a `<figure>` element, as that's the only allowed parent of a [`<figcaption>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption):

```suggestion
            return h('figure', null, [ styledH('div', attributes), figcaption ]);
```

This is nice because it attaches more semantic meaning to the video for screen readers and the like. Note that it may require a couple of style tweaks to make it look right.